reviews/2020-Network Service Mesh-annual.md

```diff
@@ -0,0 +1,192 @@
+# Network Service Mesh Sandbox Annual Review 2020
+
+## Table of Contents
+* [Background](#background)
+* [Alignment with Cloud Native](#alignment-with-cloud-native)
+* [Highlights of Last Year](#highlights-of-last-year)
+* [Annual Review Questions](#annual-review-questions)
+* [Project Links](#project-links)
+
+## Background
+Network Service Mesh provides a Hybrid/Multi-cloud IP Service mesh. Inspired by L7 service meshes like Linkerd, Istio, 
+and others, Network Service Mesh maps the concept of a service mesh from L7 payloads to L2/L3 payloads.
+
+Network Service Mesh recognizes and respects that the existing Kubernetes Networking provides excellent service for 
```

Comment:
Is there a compatibility matrix for NetworkMesh/Kubernetes versions?

Reply:
That is an excellent question.  We have focused in our CI on hitting a large number of variations of K8s across public cloud and on-prem, with patch by patch CI for:
* Vanilla K8s on bare metal (at Packet.net)
* GKE
* EKS
* AKS

it sounds like you are looking for a compatibility matrix by K8s release version?  Am I understanding you correctly?

Reply:
@edwarnicke yup, correct, k8s release versions.

Reply:
@alena1108 I've added a compatibility matrix for the [NSM v0.0.2 release](https://networkservicemesh.io/docs/releases/b/).  Short version from that page is:

> NSM is extremely insensitive to changes in Kubernetes, and as such is generally expected to work with post 1.11 versions of Kubernetes without issue.
> 
> NSM v0.0.2 has been successfully integration tested with Kubernetes versions:
> 
> - 1.12.10
> - 1.13.12
> - 1.14.10
> - 1.15.11
> - 1.16.2, 1.16.15
> - 1.17.11
> - 1.18.8
> - 1.19.0
> 
> Note: Some Kubernetes versions listed here are more recent than NSM v0.0.2 because NSM v0.0.2 has been tested with new Kubernetes versions as they have been released to verify continued compatibility.
>